RetailCoder.VBE/Refactorings/ExtractInterface/ExtractInterfacePresenterFactory.cs

```diff
@@ -32,14 +32,7 @@ public ExtractInterfacePresenter Create()
             }
 
             var model = new ExtractInterfaceModel(_state, selection.Value);
-            if (!model.Members.Any())
-            {
-                // don't show the UI if there's no member to extract
```

Comment:
this comment explains why we return `null`. I'd opt to keep it

Reply:
Do note that I dislike not showing the UI on a "user error"

Reply:
@Vogel612 agreed, needs a messagebox or something.